solara/website/pages/documentation/components/layout/griddraggable.py

```diff
@@ -33,30 +33,29 @@ def Page():
 
     # some placeholders
     items = [ColorCard(title=f"Child {i}", color=colors[i]) for i in range(len(grid_layout))]
-    with solara.VBox() as main:
-        resizable = solara.ui_checkbox("Allow resizing", value=True)
-        draggable = solara.ui_checkbox("Allow dragging", value=True)
 
-        def reset_layout():
-            set_grid_layout(grid_layout_initial)
+    resizable = solara.ui_checkbox("Allow resizing", value=True)
+    draggable = solara.ui_checkbox("Allow dragging", value=True)
```

Comment:
I think these two should be replaced by `solara.Checkbox` and explicit state control using `solara.use_reactive`